apps/experiments/views/experiment.py

```diff
@@ -361,6 +365,7 @@ def form_valid(self, form):
         experiment = get_object_or_404(Experiment, team=self.request.team, pk=self.kwargs["pk"])
         file = super().form_valid(form)
         experiment.files.add(file)
+        store_rag_embedding(experiment)
```

Comment:
To execute this as a task you should call `.delay`:
```suggestion
        store_rag_embedding(experiment.id)
```

Also we just pass the experiment ID to avoid serialising the whole experiment object